parcels/field.py

```diff
@@ -284,15 +285,20 @@ def from_netcdf(cls, filenames, variable, dimensions, indices=None, grid=None,
             else:
                 raise RuntimeError('interp_method is a dictionary but %s is not in it' % variable[0])
 
-        with NetcdfFileBuffer(lonlat_filename, dimensions, indices, netcdf_engine, field_chunksize=False, lock_file=False) as filebuffer:
+        if netcdf_engine == 'xarray':
+            FieldFileBuffer = XarrayFileBuffer
+        else:
+            FieldFileBuffer = NetcdfFileBuffer
```

Comment:
Sorry but: please don't overwrite package- or class names with other classes. This creates massive confusion later, cause most programmers will assume `FieldFileBuffer` to be the abstract super-class, not one of its main classes.

you can put that in a variable, e.g.:
```
self._field_fb_class = FieldFileBuffer
self._field_fb_class = NetcdfFileBuffer if netcdf_engine is not None else self._field_fb_class
self._field_fb_class = self._field_fb_class if netcdf_engine != 'xarray'

with self._field_fb_class( ... )
```

Reply:
Fixed in ff39e0e6fa6bc54ffe14de423bea09b48d556faf